### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by yuzhigang33** [devongovett/atom-jade#39]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app/src/package.js:550:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app/src/package.js:172:43)
```

**Comment by LinusU** [comment]
Dupe of #30

**Comment by LinusU** [comment]
Fixed in 0.4.0
